content/docs/intro/console/collaboration/teams.md

```diff
@@ -16,12 +16,17 @@ The Pulumi Console offers role-based access control (RBAC) using teams.
 Teams allow organization admins to assign a set of stack permissions
 to a group of users.
 
-### Creating a Team
+## Team Roles
+
+Members of a team can be granted `Team admin` or `Team member` permissions. Team admin's can add members to a
+team. Both team admins and team members can grant stack access to a team. By default, any new team members will be
+assigned the team member role. To change a team member's role use the 3-dot menu at the end of the table row.
```

Comment:
Does three-dot mean ellipsis in this context?

```suggestion
assigned the team member role. To change a team member's role use the ellipsis menu item at the end of the table row.
```

Reply:
yes